src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -275,11 +276,11 @@ export const StyledAvatar = styled(Avatar)<ChannelProps>`
   cursor: ${({ channelClickable }) => (channelClickable ? 'pointer' : 'auto')};
 `
 
-export const TitleHeader = styled(Text)<MainProps & ScalesWithCoverProps & ClickableProps>`
+export const TitleHeader = styled(Text)<MainProps & ClickableProps & SizeProps>`
   margin: 0;
   margin-bottom: ${sizes(2)};
   font-weight: ${typography.weights.bold};
-  font-size: calc(${(props) => props.scalingFactor} * ${typography.sizes.h6});
+  font-size: ${({ size }) => (size === 'small' ? typography.sizes.h6 : typography.sizes.subtitle1)};
   ${({ main }) => main && fluidRange({ prop: 'fontSize', fromSize: '24px', toSize: '40px' })};
```

Comment:
Let's actually remove everything `main` related. We still use this variant in search results, but let's just get rid of it there.